course/assets/js/course_info_toggle.ts

```diff
@@ -11,6 +10,9 @@ export const initDesktopCourseInfoToggle = () => {
       const desktopCourseInfoToggle = document.getElementById(
         "desktop-course-info-toggle"
       )
+      if (!(desktopCourseInfoToggle && desktopCourseInfo && mainContent)) {
+        return
+      }
```

Comment:
I'm pretty sure `mainContent` should always exist; the other two do not exist on the homepage, though this code still seems to run.